### PR TITLE
Use of the PWA library in the example App

### PR DESCRIPTION
**Pull request by forkimenjeckayang**
Closes https://github.com/adorsys/eudiw-app/issues/276

By integrating the usePWA library in the example app with some extra configurations, our application can now be used as a PWA.

### CODE REVIEW

**Comment by forkimenjeckayang** [comment]
Closses: adorsys/eudiw-app#276

File: src/App.tsx [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
-import MainSection from './components/MainSection';
-import Navbar from './components/Navbar';
-import ActivitiesPage from './pages/ActivitiesPage';
-import ContactForm from './pages/ContactForm';
-import ContactsPage from './pages/ContactsPage';
-import SettingsPage from './pages/SettingsPage';
-import Wallet from './pages/Wallet';
+import { usePWA } from "@adorsys-gis/usepwa";
+import "@adorsys-gis/usepwa/dist/src/lib/components/main.css";
+import { ThemeProvider, createTheme } from "@mui/material/styles";
```
Comment (IngridPuppet) [inline src/App.tsx:3]: I'm not against importing the CSS file here, but this makes me think there is still a problem in the usePWA library. Can you update me on how you managed to fix CSS embedding from there?

File: src/App.tsx [hunk 2]
```diff
@@ -1,18 +1,22 @@
...
+import ContactsPage from "./pages/ContactsPage";
+import SettingsPage from "./pages/SettingsPage";
+import Wallet from "./pages/Wallet";
 
 // Create the theme for the app
 const theme = createTheme();
 
 function App() {
+  usePWA();
+
```
Comment (IngridPuppet) [inline src/App.tsx:19]: What is the effect of calling this hook without doing anything of the returned value?

File: tsconfig.json [hunk 3]
```diff
@@ -23,5 +23,5 @@
   },
   "include": [
     "src"
-  ]
+, "public/service-worker.js"  ]
```
Comment (IngridPuppet) [inline tsconfig.json:26]: Why add this? It is not even a TS file.

File: src/index.tsx [hunk 4]
```diff
@@ -1,12 +1,52 @@
...
\ No newline at end of file
+
+function TempApp() {
+  return (
+    <Router>
+      {/* Wrap App with InstallPWAContextProvider */}
+      <InstallPWAContextProvider
+        component={isIosOrSafariDesktop() ? "tooltip" : "banner"}
+        installPromptTimeout={30000}
+      >
```
Comment (IngridPuppet) [inline src/index.tsx:20]: I'm not prompted to install the PWA. Can you debug my environment whenever possible?

**Review by IngridPuppet — CHANGES_REQUESTED** [review]
The `usePWA` library should be compatible with minor updates to React 18.
```json
  "peerDependencies": {
    "react": "^18.2.0",
    "react-dom": "^18.2.0"
  },
```

**Review by IngridPuppet — APPROVED** [review]

**Pull request merged**